packages/relay/test/utils/UserStateFactory.ts

```diff
@@ -11,19 +11,22 @@ export class UserStateFactory {
     prover: Prover
     unirepAddress: string
     attesterId: string
+    synchronizer: Synchronizer
 
     constructor(
         db: DB,
         provider: ethers.providers.JsonRpcProvider,
         prover: Prover,
         unirepAddress: ethers.Contract,
-        attesterId: ethers.Contract
+        attesterId: ethers.Contract,
+        synchronizer: Synchronizer
```

Comment:
it should be UnirepSocialSynchronizer

Reply:
Use `Synchronizer` is fine I think, we could use this argument as generic class.